app/controllers/application_controller.rb

```diff
@@ -983,15 +982,30 @@ def view_to_hash(view, fetch_data = false)
     view_context.instance_variable_set(:@explorer, @explorer)
     table.data.each do |row|
       target = @targets_hash[row.id] unless row['id'].nil?
-      if fetch_data && defined?(@gtl_type) && @gtl_type != "list"
-        quadicon = view_context.render_quadicon(target) if !target.nil? && type_has_quadicon(target.class.name)
+      if fetch_data && defined?(@gtl_type) && @gtl_type != "list" && !target.nil? && type_has_quadicon(target.class.name)
+        policy_sim = @lastaction == "policy_sim" && !!@policy_sim
```

Comment:
Please don't add the `lastacion == "policy_sim"` this should be handeled by `gtl_type` in coresponding controller.

Reply:
@karelhala I copied your code :trollface:

Reply:
@skateman but that code was not working with `gtl_type`, so you don't need to copy it here.